inst/examples/e2e-test/R/shinyInReact.R

```diff
@@ -50,7 +54,8 @@ shinyInReactServer <- function(id) {
       updateTextField.shinyInput(session, ns("textInputButton"), value = "new text")
     })
     
-    ids <- c("textInput", "textAreaInput", "checkboxInput", "textInputHidden", "textInputButton") 
+    ids <- c("textInput", "textAreaInput", "checkboxInput",
+             "textInputHidden", "textInputButton", "textInputShiny") 
```

Comment:
Please break lines like this:
```suggestion
    ids <- c(
      "textInput", "textAreaInput", "checkboxInput",
      "textInputHidden", "textInputButton", "textInputShiny"
    )
```